jenkins/scripts/files/run_fetch_logs.sh

```diff
@@ -81,6 +81,15 @@ sudo sh -c "cp -r /var/log/atop/* ${LOGS_DIR}/metrics/atop/"
 sudo sh -c "cp -r /var/log/sysstat/* ${LOGS_DIR}/metrics/sysstat/"
 sudo chown -R "${USER}:${USER}" "${LOGS_DIR}/metrics"
 
+# Fetch BML log if exists
+if [ -d "/tmp/BMLlog" ]; then
+  mkdir -p "${LOGS_DIR}/BML_serial_logs/"
+  cp -r /tmp/BMLlog/. "${LOGS_DIR}/BML_serial_logs/"
```

Comment:
Should we also `rm -rf /tmp/BMLlog` or is it cleaned up automatically? I'm thinking since it is the BML it will not start from scratch each time.

Reply:
I agree to remove the logs from tmp folder as it will not be necessary.

Reply:
We do have automatic cleanup steps before starting the tests [here](https://github.com/metal3-io/project-infra/blob/main/jenkins/scripts/bare_metal_lab/deploy-lab.yaml#L56-L80) so removing of the serial logs folder from previous run can be added to that list